.github/integration/scripts/values.yaml

```
@@ -0,0 +1,164 @@
+# Default values for htsget-rs.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: harbor.nbis.se/gdi/htsget-rs
+  pullPolicy: IfNotPresent
+  tag: ""
+
+imagePullSecrets: [] #
+nameOverride: ""     #
+fullnameOverride: "" #
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # # runAsUser: 1000
+  # seccompProfile:
+  #           type: "RuntimeDefault"
+
+service:
+  type: ClusterIP
+  port: 8080
+
+ingress:
+  enabled: true
+  className: "nginx"
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
```

```suggestion
    # cert-manager.io/cluster-issuer: 
```

Better to replace these with something that is not obsolete if you want some examples.